appgate/bytes.py

```diff
@@ -49,8 +49,8 @@ def certificate_bytes(value: Any, data: str) -> Entity_T:
         cert_data = {
             'version': cert.version.value + 1,
             'serial': str(cert.serial_number),
-            'issuer': cert.issuer.rfc4514_string(),
-            'subject': cert.subject.rfc4514_string(),
+            'issuer': ', '.join(cert.issuer.rfc4514_string().split(',')),
+            'subject': ', '.join(cert.subject.rfc4514_string().split(',')),
```

Comment:
`rfc4514_string` is implemented like this:
```Python
        return ",".join(
            attr.rfc4514_string() for attr in reversed(self._attributes)
        )
```

I would be more proper to define another method to create a controller compatible DN directly, something like:
```Python
        return ", ".join(
            attr.rfc4514_string() for attr in reversed(self._attributes)
        )
```
In case of issues with escape characters and such.

Reply:
:+1: